FunctionalTableData/TableSection.swift

```diff
@@ -20,7 +20,7 @@ public protocol TableSectionType {
 	var rows: [CellConfigType] { get }
 	/// Action to perform when the header view comes in or out of view.
 	var headerVisibilityAction: ((_ view: UIView, _ visible: Bool) -> Void)? { get }
-	@available(*, deprecated: 1.0, message: "Use `rows.count` instead.")
+	@available(iOS, deprecated: 1.0, message: "Use `rows.count` instead.")
```

Comment:
do we want to constraint FTD to iOS? there's nothing stopping it from working in tvOS is there?

Reply:
I mean, other than nobody giving a shit about tvOS, that is

Reply:
There are no tvOS targets right now.